packages/adp-tooling/tsconfig.eslint.json

```diff
@@ -1,4 +1,5 @@
 {
     "extends": "./tsconfig.json",
-    "include": ["src", "test", ".eslintrc.js"]
+    "include": ["src", "test", ".eslintrc.js"],
+    "exclude": []
```

Comment:
why the empty `exclude`?

Reply:
Because we are excluding something in `./tsconfig` that I don't want to be excluded here, so I am setting `exclude` to empty again.